engine/src/main/java/org/terasology/rendering/opengl/FBO.java

```diff
@@ -104,37 +104,39 @@ public void bind() {
      * Meanwhile shaders might output not just colors but additional per-pixel data. This method establishes on which
      * of an FBOs attachments, subsequent opengl commands and shaders will draw on.
      *
-     * @param c If True the color buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
-     * @param n If True the normal buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
-     * @param l If True the light buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     * @param color If True the color buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     * @param normals If True the normal buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     * @param lightBuffer If True the light buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
      */
-    public void setRenderBufferMask(boolean c, boolean n, boolean l) {
-        if (this.color == c && this.normal == n && this.lightBuffer == l) {
+    public void setRenderBufferMask(boolean color, boolean normals, boolean lightBuffer) {
```

Comment:
I couldn't make the arguments' name same as field names, because of CheckStyle rules. 

Regarding the rename, I think its much more clearer than just for example "color". Let me make rename this in my next commit.